src/pgduckdb_duckdb.cpp

```diff
@@ -202,6 +198,8 @@ DuckDBManager::LoadExtensions(duckdb::ClientContext &context) {
 		if (extension.name == "httpfs") {
 			continue;
 		}
+
+		DuckDBQueryOrThrow(context, "INSTALL " + extension.name);
```

Comment:
Interesting, was this not needed before?

Reply:
In the test we insert rows directly in `extensions` table, so we don't run the `install_extension` function. I hesitated using the function or making sure that no matter what we install the extension if needed as it makes the UX nicer, though I don't have strong opinion on it.

Reply:
Is `INSTALL` an actual no-op if the extension is already installed? Or does it do something semi-expensive like network requests, which would be good to avoid on every DB initialization.

Reply:
Yep it is a no-op, cf. [here](https://github.com/duckdb/duckdb/blob/52b43b166091c82b3f04bf8af15f0ace18207a64/src/main/extension/extension_install.cpp#L538-L548)